providers/openmetrics/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/grpc-ecosystem/go-grpc-middleware/providers/openmetrics/v2
+
+go 1.15
+
+require (
+	github.com/golang/protobuf v1.4.3
+	github.com/grpc-ecosystem/go-grpc-middleware/v2 v2.0.0-rc.2
```

Comment:
Let's update this to commit that does have your change ;p 

See similar discussion here: https://cloud-native.slack.com/archives/CL25937SP/p1611245939002300

Reply:
I think I need to add the interceptor change in a different PR, and then tag that commit.

Reply:
Still you need to put newest commit here not v2.0.0-rc.2

Reply:
Yeah forgot it before, have added it now 😁